cosmwasm/packages/wasmi-runtime/src/consts.rs

```diff
@@ -48,9 +48,12 @@ pub const MRSIGNER: [u8; 32] = [
 #[cfg(feature = "production")]
 pub const SIGNING_METHOD: SigningMethod = SigningMethod::MRENCLAVE;
 
-#[cfg(not(feature = "production"))]
+#[cfg(all(not(feature = "production"), not(feature = "test")))]
 pub const SIGNING_METHOD: SigningMethod = SigningMethod::MRENCLAVE;
```

Comment:
This is weird - it says "if compiling for neither production, nor test". what does that leave? does this just represent a default?

Reply:
Hardware mode that isn't running tests